csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
-
-# Make sure that IE can download the attachments under https.
-header( 'Pragma: public' );
-
-header( 'Content-Type: text/csv; name=' . urlencode( file_clean_name( $t_filename ) ) );
-header( 'Content-Transfer-Encoding: BASE64;' );
-
-# Added Quotes (") around file name.
-header( 'Content-Disposition: attachment; filename="' . urlencode( file_clean_name( $t_filename ) ) . '"' );
+csv_emit_headers( $t_filename );
```

Comment:
> I doubt most users understand what this is about

Sure, but they will have the option to try if there is a problem.

> Or maybe we can have "Export to CSV" and "Export to CSV (UTF-8)"

Sounds good as the end user can decide.